src/Bundler.js

```diff
@@ -332,14 +335,28 @@ class Bundler extends EventEmitter {
       let thrown = err;
 
       if (thrown.message.indexOf(`Cannot find module '${dep.name}'`) === 0) {
+        let isLocalFile = dep.name.startsWith('.');
+        // Attempt to install missing npm dependencies
```

Comment:
Mind adding tests for this?

Reply:
Yep. I’ve tried a few times but I’m gonna need to wait for #881 to land. Atm the second I add tests which use the test `/input` directory it causes a bunch of other flaky tests to go crazy.

Reply:
You'll need something like `let isLocalFile = /^[/~.]/.test(dep.name);` in order to support the new resolver. See https://github.com/parcel-bundler/parcel/blob/master/src/Resolver.js#L88-L114

Reply:
Done